.buildkite/commands/configure-environment.sh

```diff
@@ -0,0 +1,4 @@
+.buildkite/commands/configure-git-for-release-management.sh
+
+restore_cache "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+install_gems
```

Comment:
Have you considered adding a shebang here, for consistency but also so that if `configure-git-for...` fails, this script will fail as well?

```suggestion
#!/bin/bash -eu

.buildkite/commands/configure-git-for-release-management.sh

restore_cache "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
install_gems
```

Reply:
Added the shebang in https://github.com/woocommerce/woocommerce-ios/commit/4713ecf8b901f02f68557b5b259a43a223a159ac